_data/tool_and_resource_list.yml

```diff
@@ -1290,3 +1290,407 @@
   id: camflow
   name: camflow
   url: https://camflow.org/
+- description: The R package will perform data quality checks against an OMOP CDM instance.
+  id: ohdsi
+  name: OHDSI
+  url: https://github.com/OHDSI/DataQualityDashboard
+- description: Tool to classify the taxonomic origin of a read in pathogen sequencing data
+  id: centrifuge
+  name: Centrifuge
+  url: https://bio.tools/centrifuge
+- description: R package for cell-type-specific co-expression inference from single cell RNA-sequencing data.
+  id: cs-score
+  name: CS_Score
+  url: https://github.com/ChangSuBiostats/CS-CORE
+- description: Tool to remove human reads from pathogen sequencing data.
+  id: deconseq
+  name: DeconSeq
+  url: https://bio.tools/deconseq
+- description: Tool to remove human reads from pathogen sequencing data
+  id: gencof
+  name: GenCoF
+  url: https://github.com/MattCzajkowski/GenCoF
+- description: Tool to classify the taxonomic origin of a read in pathogen sequencing data.
+  id: kraken
+  name: Kraken
+  url: https://bio.tools/kraken
+- description: Tool to remove human reads from pathogen sequencing data.
+  id: metagenie
+  name: MetaGeniE
+  url: https://github.com/ngsclinical/metagenie
+- description: Sample quality control. Infer family connections thus revealing any sample mixup during sample collection from families.
+  id: peddy
+  name: PEDDY
+  url: https://github.com/brentp/peddy
+- description: The package naniar is used for exploring missing data structures with minimal deviation from the common workflows of ggplot and tidy data (Wickham, 2014, Wickham, 2009).
+  id: naniar
+  name: naniar
+  url: https://cran.r-project.org/web/packages/naniar/vignettes/getting-started-w-naniar.html
+- description: missingno provides a small toolset of flexible and easy-to-use missing data visualizations and utilities that allows you to get a quick visual summary of the completeness (or lack thereof) of your dataset.
+  id: missingno
+  name: missingno
+  url: https://github.com/ResidentMario/missingno
+- description: A collection of tools that support data diagnosis, exploration, and transformation.
+  id: dlookr
+  name: dlookr
+  url: https://cran.r-project.org/web/packages/dlookr/dlookr.pdf
+- description: Declare data validation rules and data quality indicators; confront data with them and analyze or visualize the results.
+  id: validate
+  name: validate
+  url: https://cran.r-project.org/web/packages/validate/validate.pdf
+- description: Validate dataset by columns and rows using convenient predicates inspired by 'assertr' package.
+  id: data-validator
+  name: data.validator
+  url: https://cran.r-project.org/web/packages/data.validator/data.validator.pdf
```

Comment:
I would prefer that we link to the GitHub rpeo of tools and not the instructions, if possible. For example: https://github.com/Appsilon/data.validator/